### ISSUE TITLE
ECO trained the test dataset to get high verification accuracy？！

### ISSUE THREAD

**Issue by PPPxiaomoer** [zhang-can/ECO-pytorch#42]
Hello, thank you for your contribution in the pytorch-ECO code in GitHub. However, I found some problems in the validate function of main.py.

It looks like 254 lines in main.py：
 # compute output
output = model(input_var)
loss = criterion(output, target_var)

It seems that the ECO network trained the validation data set. I hope you take the time to check it out during your busy schedule.
